kong/plugins/jwt/handler.lua

```diff
@@ -176,11 +176,7 @@ function JwtHandler:access(conf)
   JwtHandler.super.access(self)
 
   -- check if preflight request and whether it should be authenticated
-  if conf.run_on_preflight == false and get_method() == "OPTIONS" then
-    -- FIXME: the above `== false` test is because existing entries in the db will
-    -- have `nil` and hence will by default start passing the preflight request
-    -- This should be fixed by a migration to update the actual entries
-    -- in the datastore
+  if get_method() == "OPTIONS" and not conf.run_on_preflight then
```

Comment:
If we check for `conf.run_on_preflight` first, we can avoid calling `get_method()` needlessly?